util/types/mydecimal_test.go

```diff
@@ -297,6 +297,83 @@ func (s *testMyDecimalSuite) TestRound(c *C) {
 	doTest(c, tests)
 }
 
+func (s *testMyDecimalSuite) TestRoundWithTruncate(c *C) {
+	type tcase struct {
```

Comment:
It seems the style of this file, I'm not pretty sure whether we should keep consistent with the above tests. Maybe we could update the style in another independent PR == @ngaut